dev/com.ibm.ws.microprofile.reactive.streams.operators/src/com/ibm/ws/microprofile/reactive/streams/operators/spi/impl/WASReactiveStreamsEngineImpl.java

```diff
@@ -57,9 +57,22 @@
     public void activate(ComponentContext cc) {
         executorServiceRef.activate(cc);
         ReactiveStreamsFactoryResolver.setInstance(new ReactiveStreamsFactoryImpl());
+        ReactiveStreamsEngineResolver.setInstance(this);
```

Comment:
I think this is related to what Glenn saw. As I saw the same thing afterwards and had to change some of my tests and never went back and looked at it.